internal/cli/cmd/config.go

```diff
@@ -80,12 +82,33 @@ func newCmdConfigSet() *cobra.Command {
 			if err := check.Set(context.Background(), args[0], args[1]); err != nil {
 				fatalf("Unable to set config:  %s", err)
 			}
+			if !params.Restart {
+				fmt.Println("⚠️  Restart Cilium pods for configmap changes to take effect")
+				return nil
+			}
+			if err := k8sClient.DeletePodCollection(context.Background(), params.Namespace, metav1.DeleteOptions{}, metav1.ListOptions{LabelSelector: defaults.CiliumPodSelector}); err != nil {
+				fmt.Printf("⚠️  Unable to restart Cilium pods: %s\n", err)
+			} else {
+				fmt.Println("♻️  Restarted Cilium pods")
+			}
 			return nil
 		},
 	}
 
-	cmd.Flags().StringVarP(&params.Namespace, "namespace", "n", "kube-system", "Namespace Cilium is running in")
```

Comment:
We use the single line formatting in all other files in `internal/cli/cmd`. I think we should stay consistent across the code base. If we decide to use multi line formatting, let's change all other occurences too.

Reply:
I'm happy to change other occurrences in this file since I've already made similar changes. As for other files, I'm inclined to defer it until we have a consensus around the formatting structure.

Reply:
I think in this case we should stick to the single line version in this file as well for this PR and - if we decide that we want to change the format - change all occurrences in the code at once in a follow-up PR.

Reply:
Why is that blocking for this PR?

Reply:
Because this PR actively changes the formatting and the formatting is IMO unrelated to the functionality being introduced.

Reply:
Correct, I'm changing the ones in the file that I'm touching. 

>I dont' think this was caused by gofmt, right?

That's right. I've made the change for improving readability. But this is a good point. I don't see any opinionated comments in the official go documentation. Do you know of any, either for or against such a formatting change?

I was able to find a related GitHub issue for the "stricter fmt" in favor of this formatting style - https://github.com/mvdan/gofumpt/issues/2.

Reply:
> Correct, I'm changing the ones in the file that I'm touching.
> 
> > I dont' think this was caused by gofmt, right?
> 
> That's right. I've made the change for improving readability. But this is a good point. I don't see any opinionated comments in the official go documentation. Do you know of any, either for or against such a formatting change?

I don't know whether there are any "official" Go guidelines. Personally, I prefer for the options to stay single line as it allows for an easy `git grep cmd\.Flags` to get an overview of options with their name and description from the code. Though, I agree that readability might suffer a bit depending on line length.

My review was primarily on the point that we shouldn't switch single occurrence to a multi-line format while keeping others in the existing single-line format. IMO we should strive to stay consistent for cases like these where `gofmt` doesn't enforce a particular formatting.

> I was able to find a related GitHub issue for the "stricter fmt" in favor of this formatting style - [mvdan/gofumpt#2](https://github.com/mvdan/gofumpt/issues/2). Looks like they finally supported the multi-line feature - [mvdan/gofumpt#2 (comment)](https://github.com/mvdan/gofumpt/issues/2#issuecomment-796878776)

Interesting, will check this out.

Reply:
Let's stick to the existing notion. Single line. We use it in the entire codebase.